lattice.schema.json

```diff
@@ -0,0 +1,51 @@
+{
+    "$schema": "https://json-schema.org/schema#",
+    "$id": "https://github.com/bonnkleiford/MSPLib-Library/lattice.schema.json",
+    "title": "The schema for MSPFormat representation of an MSP lattice.",
+    "type": "object",
+    "required": ["nodeNumber"],
```

Comment:
I don't think this is correct. It implies that the top-level is `{"nodeNumber": {...}}`. The top-level should just be `additionalProperties` if you want `{0: {...}, 1: {...}, 2: {...}}` and so on.